umh.docs.umh.app/content/en/docs/production-guide/installation/flatcar-installation-bare-metal.md

```diff
@@ -13,106 +13,162 @@ aliases:
 
 Here is a step-by-step guide on how to deploy the UMH stack on
```

Comment:
![image](https://github.com/united-manufacturing-hub/umh.docs.umh.app/assets/30755722/4e7d02a6-6133-4f17-9657-534cad1b9e50)

can you reduce the wall of text?

Reply:
maybe just put it into 2 paragraphs